### PR TITLE
fixing link to issue template and vs code image 

### PR DESCRIPTION
**Pull request by helper-uttam**
Signed-off-by:helper-uttam <bajiraouttamsinha@gmail.com>

**Description**
Sometime Vs code image is not showing up due to broken link and there is also a wrong link to the issue template.

**Notes for Reviewers**
- [x] Fixing the link to issue template.
- [x] Fixing link to the image of VScode-sign-off that doesn't render sometime.

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by YashKamboj** [comment]
@warunicorn19 Views??

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -269,6 +269,6 @@ To ensure all your commits are signed, you may choose to add this alias to your
 
 Or you may configure your IDE, for example, Visual Studio Code to automatically sign-off commits for you:
 
-<a href="./.github/assets/images/git-signoff-vscode.png" ><img src="./.github/assets/images/git-signoff-vscode.png" width="50%"/><a>
```
Comment (YashKamboj) [inline CONTRIBUTING.md:272]: I am not sure if the link is broken?

Comment (helper-uttam) [reply CONTRIBUTING.md:272]: > I am not sure if this link is broken?

I found several time that the pic is not showing up. So I thought of refactoring it! 
Let me know if it doesn't looks good.

Comment (YashKamboj) [reply CONTRIBUTING.md:272]: It rendered  perfectly every time for me, btw how do you checked that it is working now? @helper-uttam

Comment (helper-uttam) [reply CONTRIBUTING.md:272]: While editing I checked from preview tab, it was working.

Comment (YashKamboj) [reply CONTRIBUTING.md:272]: i am not sure , i think the previous link is correct 🤔🤔

Comment (helper-uttam) [reply CONTRIBUTING.md:272]: Okay will update it tomorrow! 
Thanks for reviewing.

Comment (YashKamboj) [reply CONTRIBUTING.md:272]: wait, let @Abhijay007 review it, then change the link to the previous one (i may be wrong)

Comment (Abhijay007) [reply CONTRIBUTING.md:272]: it is working fine for me too, I think no need to change that link, kindly revert this change !!

**Comment by helper-uttam** [comment]
image link reverted

**Review by YashKamboj — APPROVED** [review]

**Review by leecalcote — CHANGES_REQUESTED** [review]
The link to create a new issue is ideal vs. the link that shows the issue template.

**Comment by leecalcote** [comment]
Thanks for contributing, @helper-uttam. Let's close out this PR as it currently has no desired changes. Here's a link to all the other `help wanted` issues for convenience - https://github.com/issues?q=is%3Aopen+is%3Aissue+archived%3Afalse+org%3Alayer5io+org%3Ameshery+org%3Aservice-mesh-performance+org%3Aservice-mesh-patterns+label%3A%22help+wanted%22+

**Pull request closed without merging**